src/lib/components/License/LicenseField.js

```diff
@@ -44,10 +71,29 @@ class LicenseFieldForm extends Component {
               const arrayPath = fieldPath;
               const indexPath = index;
               const key = `${arrayPath}.${indexPath}`;
+              const uiKey = `${uiFieldPath}.${indexPath}`;
               const licenseType = value.id ? 'standard' : 'custom';
-              const description = getIn(values, `${key}.description`, '');
-              const link = getIn(values, `${key}.link`, '');
-              const title = getIn(values, `${key}.title`, '');
+              const description = getIn(
```

Comment:
Here now we need to check for the UIobject for any already loaded license but we need to rely in the value inside "metadata" for the added values from the interface, I cannot find a solution to only use the UIObject.